modules/loader-utils/src/lib/files/node-file-facade.ts

```diff
@@ -25,7 +25,7 @@ export class NodeFileFacade implements ReadableFile, WritableFile {
     throw new Error('Can\'t instantiate NodeFile. Make sure to import @loaders.gl/polyfills first.');
   }
   /** Read data */
-  async read(start?: number | bigint, end?: number | bigint): Promise<ArrayBuffer> {
+  async read(start?: number | bigint, offset?: number): Promise<ArrayBuffer> {
```

Comment:
- What is the difference between `start` and `offset`? 
- How does this function now know where to stop reading?
- Should it be `length` instead?

Reply:
That's my mistake, I'll change it to `length`, as it's in the `ReadableFile` and `NodeFile` implementations